Model/Layer/Filter/Stock.php

```diff
@@ -167,4 +174,33 @@ public function getProductsCount($value)
         );
         return $collection->getConnection()->fetchOne($select);
     }
+
+    /**
+     * Checks if the search engine is currently configured to use any version of Elasticsearch.
+     * @return bool
+     */
+    private function isSearchEngineElasticsearch(): bool
+    {
+        $searchEngine = $this->_scopeConfig->getValue('catalog/search/engine');
+
+        return strpos($searchEngine, 'elasticsearch') !== false;
+    }
+
+    /**
+     * Get url param name
+     * @return string
+     */
+    private function getUrlParamName()
```

Comment:
Redundant function as this class already has $this->_requestVar initialized in the same way in constructor